OCABundles/schema/bcgov-digital-trust/pilot/README.md

```diff
@@ -0,0 +1,25 @@
+# Digital Identity Trust Program Pilot Invitation
+
+A verifiable credential schema for a Pilot Invitation credential to gain access to the Person Credential.
+
+- Publishing Organization: Digital Trust, Government of British Columbia
+- Primary Contact Email Address: DItrust@gov.bc.ca
+
+## Identifiers
+
+| Identifier                                | Location  | URL                                                   |
+| ----------------------------------------- | --------- | ----------------------------------------------------- |
+| Mp2pDQqS2eSjNVA7kXc8ut:2:BC VC Pilot Certificate:1.0.1 | candy:dev | https://candyscan.idlab.org/tx/CANDY_DEV/domain/28717 |
+| 4zBepKVWZcGTzug4X49vAN:2:BC VC Pilot Certificate:1.0.1  | candy:test | https://candyscan.idlab.org/tx/CANDY_TEST/domain/30 |
+| E2h4RUJxyh48PLJ1CtGJrq:2:BC VC Pilot Certificate:1.0.1 | candy:prod | https://candyscan.idlab.org/tx/CANDY_PROD/domain/21 |
```

Comment:
Should you be using the same OCA Bundle for Dev/Test and Prod?  Wouldn’t the watermark be different between the two?